packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -29,25 +30,25 @@ export default {
   decorators: [withDesign],
 } as ComponentMeta<typeof FilterMultiSelect>
 
-export const DefaultKaizenSiteDemo = args => {
+export const DefaultKaizenSiteDemo: ComponentStory<
+  typeof FilterMultiSelect
+> = args => {
   const [selectedKeys, setSelectedKeys] = useState<Selection>(
     new Set(["id-fe"])
   )
 
-  const handleSelectionChange = (keys: Selection) => {
-    keys && setSelectedKeys(keys)
-  }
+  const handleSelectionChange = (keys: Selection) => setSelectedKeys(keys)
 
   return (
     <FilterMultiSelect
-      label="Engineer"
+      {...args}
       onSelectionChange={handleSelectionChange}
       selectedKeys={selectedKeys}
       items={items}
       trigger={() => (
         <FilterMultiSelect.TriggerButton
           selectedOptionLabels={getSelectedOptionLabels(selectedKeys, items)}
-          label="Engineer"
+          {...args}
```

Comment:
Hmm... This probably doesn't make sense, because not all props for `FilterMultiSelect` would apply to `TriggerButton`.

This could probably work though?

```suggestion
          label={args.label}
```